eth/executionclient/execution_client.go

```diff
@@ -264,14 +264,14 @@ func (ec *ExecutionClient) Healthy(ctx context.Context) error {
 	recordRequestDuration(ctx, ec.nodeAddr, time.Since(start))
 
 	if sp != nil {
-		recordExecutionClientStatus(ctx, statusSyncing, ec.nodeAddr)
-
 		syncDistance := max(sp.HighestBlock, sp.CurrentBlock) - sp.CurrentBlock
 
 		observability.RecordUint64Value(ctx, syncDistance, syncDistanceGauge.Record, metric.WithAttributes(semconv.ServerAddress(ec.nodeAddr)))
 
 		// block out of sync distance tolerance
 		if syncDistance > ec.syncDistanceTolerance {
+			recordExecutionClientStatus(ctx, statusSyncing, ec.nodeAddr)
```

Comment:
@oleg-ssvlabs If the code doesn't enter this `if`,  then `syncDistanceGauge` is observed twice: first with `syncDistance` value before the `if` and then with `0` in `syncDistanceGauge.Record(ctx, 0, metric.WithAttributes(semconv.ServerAddress(ec.nodeAddr)))`